package.json

```diff
@@ -176,6 +176,7 @@
     "Rob Cherry <rcherry@reverbnation.com>",
     "Rob Dodson <lets.email.rob@gmail.com>",
     "Rémi <r3mi@users.sourceforge.net>",
+    "Sage Gerard <sage@sagegerard.com>",
```

Comment:
Please remove this, these are generated automatically based on the git history when we release a new version

Reply:
Understood. Reverting.